src/Symfony/Component/Notifier/Bridge/Bandwidth/BandwidthTransport.php

```diff
@@ -0,0 +1,106 @@
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author gnito-org <https://github.com/gnito-org>
+ */
+final class BandwidthTransport extends AbstractTransport
+{
+    protected const HOST = 'messaging.bandwidth.com';
+
+    public function __construct(
+        private readonly string $username,
+        #[\SensitiveParameter] private readonly string $password,
+        private readonly string $from,
+        private readonly string $accountId,
+        private readonly string $applicationId,
+        private readonly ?string $priority,
+        HttpClientInterface $client = null,
+        EventDispatcherInterface $dispatcher = null
+    )
+    {
+        parent::__construct($client, $dispatcher);
+    }
+
+    public function __toString(): string
+    {
+        return sprintf('bandwidth://%s?from=%s&account_id=%s&application_id=%s', $this->getEndpoint(), $this->from, $this->accountId, $this->applicationId) . ($this->priority ? sprintf('&priority=%s', $this->priority) : null);
+    }
+
+    public function supports(MessageInterface $message): bool
+    {
+        return $message instanceof SmsMessage;
+    }
+
+    /**
+     * https://dev.bandwidth.com/apis/messaging/#tag/Messages/operation/createMessage
+     */
+    protected function doSend(MessageInterface $message): SentMessage
+    {
+        if (!$message instanceof SmsMessage) {
+            throw new UnsupportedMessageTypeException(__CLASS__, SmsMessage::class, $message);
+
+        }
+        $opts = $message->getOptions();
+        $options = $opts ? $opts->toArray() : [];
+        $options['text'] = $message->getSubject();
+        $options['from'] = $options['from'] ?? $this->from;
+        $options['to'] = $options['to'] ?? [$message->getPhone()];
+        $options['account_id'] = $options['account_id'] ?? $this->accountId;
+        $options['applicationId'] = $options['application_id'] ?? $this->applicationId;
+        unset($options['application_id']);
+
+        if (!isset($options['priority']) && $this->priority) {
+            $options['priority'] = $this->priority;
+        }
+
+        if (!preg_match('/^\+[1-9]\d{1,14}$/', $this->from)) {
+            throw new InvalidArgumentException(sprintf('The "From" number "%s" is not a valid phone number. The number must be in E.164 format.', $this->from));
+        }
+
+        if (!preg_match('/^\+[1-9]\d{1,14}$/', $message->getPhone())) {
+            throw new InvalidArgumentException(sprintf('The "To" number "%s" is not a valid phone number. The number must be in E.164 format.', $message->getPhone()));
+        }
+        $endpoint = sprintf('https://%s/api/v2/users/%s/messages', $this->getEndpoint(), $options['account_id']);
+        unset($options['accountId']);
+
+        $response = $this->client->request('POST', $endpoint, ['auth_basic' => $this->username . ':' . $this->password, 'json' => array_filter($options)]);
```

Comment:
```suggestion
        $response = $this->client->request('POST', $endpoint, [
            'auth_basic' => $this->username . ':' . $this->password,
            'json' => array_filter($options),
        ]);
```

Lets make this line more readable

Reply:
:face_exhaling: :face_with_head_bandage:

This has been an extremely painful exercise. Not conducive to enthusiastic contribution.

RUN AUTOMATED CODE FORMATTING DURING MERGE OR SOMETHING LIKE THAT, if you guys want to enforce a specific code formatting standard. I'm all for it, but don't burden the contributors with it.